### ISSUE TITLE
BUG "Something went wrong": Last admitted to advanced filter on patient list

### ISSUE THREAD

**Issue by aparnacoronasafe** [coronasafe/care_fe#3482]
**Describe the bug**
"Something went wrong error comes up when "last admitted to" filter is set at ICU, on patient list
**To Reproduce**

1. Go to Patient list https://care.coronasafe.in/patients?page=2&limit=12&district=&lsgBody=&facility=&facility_type=&date_declared_positive_before=&date_declared_positive_after=&date_of_result_before=&date_of_result_after=&created_date_before=&created_date_after=&modified_date_before=&modified_date_after=&date_of_result=&last_consultation_admission_date_before=&last_consultation_admission_date_after=&last_consultation_discharge_date_before=&last_consultation_discharge_date_after=&ordering=&category=&gender=&disease_status=&age_min=&age_max=&last_consultation_admitted_to_list=&srf_id=&number_of_doses=&covin_id=&is_kasp=&is_declared_positive=&last_consultation_symptoms_onset_date_before=&last_consultation_symptoms_onset_date_after=&last_vaccinated_date_before=&last_vaccinated_date_after=&last_consultation_is_telemedicine=&is_antenatal=
2. Click on advanced filter
3. set last admitted to filter at ICU/Bed with oxygen support and apply the filter
4. See error "something went wrong appear on tp right corner of screen

**Expected behavior**
the error must not appear

**Screenshots**
<img width="1151" alt="Screenshot 2022-08-26 at 8 54 11 PM" src="https://user-images.githubusercontent.com/67050716/186939340-90e8a4c6-964d-4453-b5d2-3bca26badb92.png">

**Desktop (please complete the following information):**

- OS: MacOS
- Browser chrome,

**Comment by khavinshankar** [comment]
@aparnacoronasafe it seems that `admitted_to` field was removed at the backend, so we are getting this error

should we remove this filter at the frontend?

**Comment by Ashesh3** [comment]
Seems duplicate of https://github.com/coronasafe/care_fe/issues/2642

PR Fixing this: https://github.com/coronasafe/care_fe/pull/2659

**Comment by nihal467** [comment]
@aparnacoronasafe it's been here for a while, can you comment on it

**Comment by aparnacoronasafe** [comment]
@khavinshankar It's not that bed types are altogether removed from care. Earlier we had a more generic bed categorization. Which was removed to introduce the new "Bed Types" as an attribute of any bed within a location. 

The last admitted to filter must be updated accordingly.

@gigincg please comment

The bedtype filter is a very useful one and must not be removed completely

@nihal467 lets discuss in the EOD. @Ashesh3 @khavinshankar @vigneshhari to participate

**Comment by nihal467** [comment]
@khavinshankar once the PR is merged, close this issue
